.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,60 @@
+---
+name: New Component
+about: Suggest or propose a new component to be added to Cauldron
+title: ''
+labels: 'rfc'
+assignees: '@deque/cauldron-team,@deque/cauldron-codeowners'
+---
+
+# [component name]
+
+## Overview
+
+<!-- Provide a brief summary of the proposed component: -->
+
+### Problem Statement
+
+<!--
+  Explain why this component is necessary, considering:
+  - What problem or problems is this component trying to solve?
+  - Are there existing patterns in Deque products that this component would replace?
+  - Are there similar components in Cauldron that this would use or build upon?
+-->
+
+## Component Design
+
+### Visual Design
+
+<!-- Share any images, wireframes, or prototypes that visually demonstrate the component: -->
+
+### Interface / Props
+
+<!-- Provide example typescript interfaces necessary to understand the component's api including any required or default props: -->
+
+### Accessibility
+
+<!-- 
+  Identify any key accessibility aspects, considering:
+  - What aria properties (if any) will this component need to implement?
+  - What semantic elements will this component need to use?
+-->
+
+### Responsiveness
+
+<!-- Are there any responsiveness considerations that need to be made? This could include things like text wrapping or container queries. -->
+
+### Interactive States (optional)
+
+<!-- Does the component require any interactive states? If so, explain: -->
+
+### Variants (optional)
+
+<!-- Does the component require different variants implementations? If so, explain: -->
+
+### Usage Examples
+
+<!-- Provide usage examples of the component and/or sub-components using different props or children: -->
+
+### Referencing Implementations
```

Comment:
I think this could be optional.

Reply:
I'm not sure it should be. We shouldn't "optionally" link to different implementations, I think that should be a requirement. If there's none that exist, we can call that out.

Reply:
In that case, should there be an implementation link referenced in https://github.com/dequelabs/cauldron/issues/1812?

Reply:
If there's some that we can find, ideally yeah. Although that issue was created before this template was implemented so...